tests/testPupilCoordinates.py

```diff
@@ -1,3 +1,7 @@
+from __future__ import division
+from builtins import zip
+from builtins import range
+from past.utils import old_div
```

Comment:
I took out the old _div in the actual classes, but there were so many in
the unit tests I didn't. That's the difference.

On Fri, Aug 12, 2016, 3:22 PM danielsf <notifications@github.com> wrote:

> In tests/testPupilCoordinates.py
> <https://github.com/lsst/sims_utils/pull/23#discussion_r74668987>:
>
> > @@ -1,3 +1,7 @@
> > +from __future__ import division
> > +from builtins import zip
> > +from builtins import range
> > +from past.utils import old_div
>
> Okay. That's weird. I looked at this PR this morning and saw no old_div
> occurrences. Now, however, when I look at the PR, I see everything you're
> seeing. Not sure how to explain that.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/lsst/sims_utils/pull/23/files/495c9472247a8bb4d0bbfad429b74485d03d69b1..421cce6eaeaff719a3756511c7a2871d71e36717#r74668987>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABxQ3_FIGaUx6JR7kb2KTD71Tve83p2yks5qfPIUgaJpZM4Jircy>
> .
>